use-latest-deps-android.sh

```diff
@@ -57,26 +107,32 @@ REPO=$1
 # http://stackoverflow.com/a/246128/101923
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-set -e
+set +e
 set -x
 
-# Generate JSON dependencies report
-./gradlew dependencyUpdates -Drevision=release -DoutputFormatter=json
+generate_dependencies_report "${DIR}/"
+update_gradle_exit_code="$?"
```

Comment:
nit: name this `generate_dependencies_report_exit_code` or something else that is more similar to the actual function name?